README.md

```diff
@@ -26,7 +26,10 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose person
+  puts "Would you take this rose and help out an old beggar, person?"
```

Comment:
We want to use string interpolation here to substitute in the value of the `person` variable.

```ruby
puts "Would you take this rose and help out an old beggar, #{person}?"
```